firmware/motion_board/firmware/pic-motor-controller.X/nbproject/configurations.xml

```diff
@@ -719,7 +719,7 @@
         <property key="memories.rww" value="true"/>
         <property key="poweroptions.powerenable" value="true"/>
         <property key="programmerToGoFilePath"
-                  value="C:/Users/Dusan/HarmonyProjects/KretackaTest/firmware/KretackaTest.X/debug/default/KretackaTest_ptg"/>
+                  value="E:/eurobot2022/git/mep3/firmware/motion_board/firmware/pic-motor-controller.X/debug/default/KretackaTest_ptg"/>

```

Comment:
Jao taj generator konfiguracije sta radi :D

Reply:
To je neka stara konfiguracija, to se sve menja po potrebi :( A programmerToGo ni ne koristimo xD